.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,159 @@
+    
+- type: dropdown
+  attributes:
+    label: Core Library
+    options: 
+      - "@azure/msal or msal"
+      - "@azure/msal-browser"
+      - "@azure/msal-node"
+  validations:
+    required: true
+    
+- type: input
+  attributes:
+    label: Core Library Version
+    description: "Please enter the version you are using."
+    placeholder: "2.13.0"
+  validations:
+    required: true
+    
+- type: dropdown
+  attributes:
+    label: Wrapper Library
+    options:
+      - "@azure/msal-angular"
+      - "@azure/msal-react"
+    
+- type: input
+  attributes:
+    label: Wrapper Library Version
+    description: "Please enter the version you are using."
+    placeholder: "2.0.0-beta.0"
+    
+- type: textarea
+  attributes:
+    label: Description
+    description: "Please briefly describe your issue."
+  validations: 
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Error Message
+    description: "Please provide any error message you are receiving and a stack trace."
+    
+- type: textarea
+  attributes:
+    label: MSAL Configuration
+    description: "Please provide your MSAL configuration."
+    render: javascript
+    placeholder: |
+      {
+        auth: {
+          clientId: "clientId",
+          authority: "https://login.microsoftonline.com/common/"
+        },
+        cache: {
+          cacheLocation: "sessionStorage"
+          storeAuthStateInCookie: false
+        }
+      }
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Relevant Code Snippets
+    description: "Please provide relevant code snippets that can be used to reproduce this issue."
+    render: javascript
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Reproduction Steps
+    description: "Please provide steps to reproduce or a link to a sample which demonstrates this behavior."
+    placeholder: |
+      1.
+      2.
+      3.
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: "Please describe what you expect the behavior to be."
+  validations:
+    required: true
+    
+- type: dropdown
+  attributes:
+    label: Identity Provider
+    options: 
+      - Azure AD
+      - Azure B2C Basic Policy
+      - Azure B2C Custom Policy
+      - ADFS
+      - Other
+  validations:
+    required: true
+    
+- type: dropdown
+  attributes:
+    label: Browsers Affected (Select all that apply)
+    multiple: true
+    options: 
+      - Chrome
+      - Firefox
+      - Edge
+      - Safari
+      - Internet Explorer
+      - Other
+  validations:
+    required: true
+    
+- type: input
+  attributes:
+    label: Regression
+    description: "If this behavior worked before, please enter the last working version(s)."
+    placeholder: "@azure/msal-browser 2.0.0"
+    
+- type: dropdown
+  attributes:
+    label: Source
+    description: "Are you an internal or external customer?"
```

Comment:
Should we have optional customer product/company? May work with stats later. Just an idea.

Reply:
Will take this incrementally if we see value in it later. Right now we only care so we know whether or not we can follow up over Teams